webpages/settings/components/addon-body.html

```diff
@@ -286,6 +304,9 @@
     display: flex;
     align-items: center;
   }
+  .related-addons > .addon-message {
+    flex-wrap: wrap;
+  }
```

Comment:
Wrap is needed if the window is not wide enough to prevent the following unwanted layout:

![image](https://github.com/user-attachments/assets/a8d8fcc6-f826-47e8-b126-cecc540fe5b3)